storage-node/packages/cli/package.json

```diff
@@ -28,7 +28,7 @@
   },
   "scripts": {
     "test": "mocha 'dist/test/**/*.js'",
-    "lint": "eslint 'paths/**/*.js' 'lib/**/*.js'",
+    "lint": "eslint --ext .js,.jsx,.ts,.tsx . && tsc --noEmit --pretty",
```

Comment:
I would leave off the `&& tsc --noEmit --pretty` from the lint script, so you can add args to lint for fixing eg.

`yarn lint --fix`

I was trying to have a kind of common scripts for all packages:

lint: 'eslint ... ',
format: 'prettier . --write'
checks: 'yarn lint && prettier . --check && tsc --noEmit --pretty'